geo/Geoserver.py

```diff
@@ -625,74 +1327,106 @@ def create_featurestore(
-                <entry key="port">{4}</entry>
-                <entry key="user">{5}</entry>
-                <entry key="passwd">{6}</entry>
+                <entry key="Expose primary keys">{}</entry>
+                <entry key="host">{}</entry>
+                <entry key="port">{}</entry>
+                <entry key="user">{}</entry>
+                <entry key="passwd">{}</entry>
                 <entry key="dbtype">postgis</entry>
-                <entry key="schema">{7}</entry>
-                <entry key="database">{8}</entry>
-                <entry key="Evictor run periodicity">{9}</entry>
-                <entry key="Max open prepared statements">{10}</entry>
-                <entry key="encode functions">{11}</entry>
-                <entry key="Primary key metadata table">{12}</entry>
-                <entry key="Batch insert size">{13}</entry>
-                <entry key="preparedStatements">{14}</entry>
-                <entry key="Estimated extends">{15}</entry>
-                <entry key="fetch size">{16}</entry>
-                <entry key="validate connections">{17}</entry>
-                <entry key="Support on the fly geometry simplification">{18}</entry>
-                <entry key="Connection timeout">{19}</entry>
-                <entry key="create database">{20}</entry>
-                <entry key="min connections">{21}</entry>
-                <entry key="max connections">{22}</entry>
-                <entry key="Evictor tests per run">{23}</entry>
-                <entry key="Test while idle">{24}</entry>
-                <entry key="Max connection idle time">{25}</entry>
-                <entry key="Loose bbox">{26}</entry>
-                </connectionParameters>              
+                <entry key="schema">{}</entry>
+                <entry key="database">{}</entry>
+                <entry key="Evictor run periodicity">{}</entry>
+                <entry key="Max open prepared statements">{}</entry>
+                <entry key="encode functions">{}</entry>
+                <entry key="Primary key metadata table">{}</entry>
+                <entry key="Batch insert size">{}</entry>
+                <entry key="preparedStatements">{}</entry>
+                <entry key="Estimated extends">{}</entry>
+                <entry key="fetch size">{}</entry>
+                <entry key="validate connections">{}</entry>
+                <entry key="Support on the fly geometry simplification">{}</entry>
+                <entry key="Connection timeout">{}</entry>
+                <entry key="create database">{}</entry>
+                <entry key="min connections">{}</entry>
+                <entry key="max connections">{}</entry>
+                <entry key="Evictor tests per run">{}</entry>
+                <entry key="Test while idle">{}</entry>
+                <entry key="Max connection idle time">{}</entry>
+                <entry key="Loose bbox">{}</entry>
+                </connectionParameters>
```

Comment:
Why remove the order? Won't this increase the likelihood of developer error? Changes to this string should be to break this up and make it more maintainable.

Reply:
This must have been an oversight. I didn't even work on the method. Thanks for pointing that out.